dojo/filters.py

```diff
@@ -1663,6 +1667,8 @@ def set_date_fields(self, *args: list, **kwargs: dict):
         self.form.fields["on"].widget = date_input_widget
         self.form.fields["before"].widget = date_input_widget
         self.form.fields["after"].widget = date_input_widget
+        date_mitigated_input_widget = forms.DateInput(attrs={"class": "datepicker", "placeholder": "YYYY-MM-DD"}, format="%Y-%m-%d")
+        self.form.fields["on"].widget = date_mitigated_input_widget
```

Comment:
```suggestion
        self.form.fields["mitigated_on"].widget = date_input_widget
        self.form.fields["mitigated_before"].widget = date_input_widget
        self.form.fields["mitigated_after"].widget = date_input_widget        
```
The `date_input_widget` could be reused from above. No need to redefine the same thing twice